.changeset/brave-dryers-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/core-web-vitals': major
+---
+
+Update endpoint and send attribution data to big query
```

Comment:
Would it be more appropriate to say we added attribution ans the data will end in a different table?